src/schema.graphql

```diff
@@ -1,8 +1,13 @@
 # import CitySubscriptionPayload, PLACE_SIZES, DateTime, PAYMENT_PROVIDER, NOTIFICATION_TYPE, CURRENCY from "./generated/database.graphql"
 
 type Query {
+  hello: String
```

Comment:
why did you add this query?

Reply:
Well... It is always nice if you can say hello to your GraphQL server. I would even move this into `graphql-yoga` by default 😏